addons/account/wizard/account_tour_upload_bill.py

```diff
@@ -55,22 +67,28 @@ def _action_list_view_bill(self, bill_ids=[]):
         }
 
     def apply(self):
+        invoice_date = fields.Date.today() - timedelta(days=12)
         purchase_journal = self.env['account.journal'].search([('type', '=', 'purchase')], limit=1)
         if self.selection == 'upload':
             return purchase_journal.with_context(default_journal_id=purchase_journal.id, default_move_type='in_invoice').create_invoice_from_attachment(attachment_ids=self.attachment_ids.ids)
         elif self.selection == 'sample':
+            context = dict(self._context)
             attachment = self.env['ir.attachment'].create({
                 'type': 'binary',
-                'name': 'INV/2020/0011.pdf',
+                'name': 'INV/%s0001' % invoice_date.strftime('%Y/%m'),
                 'res_model': 'mail.compose.message',
-                'datas': self.sample_bill_preview,
+                'datas': self.preview_invoice,
```

Comment:
Datas can not be HTML. This must be a file's binary. Ideally a pdf created from the HTML.

Reply:
tip: probably using self.env['ir.actions.report']._run_wkhtmltopdf